packages/e2e-tests/examples.json

```diff
@@ -39,5 +39,7 @@
   "node/spawn.js": "c9d0539d-fde0-4949-a9f7-acdb5b56d4a6",
   "log_points_and_block_scope.html": "ad8f5490-0b9f-4488-bd3c-4f2824bc21ce",
   "cypress/doc_inspector_styles": "aa6201d2-1e83-408b-8247-cf3d26ac421a",
-  "doc_minified_chromium.html": "5cdb2ffe-beff-4d35-9689-9242bd277f2d"
+  "doc_minified_chromium.html": "5cdb2ffe-beff-4d35-9689-9242bd277f2d",
+  "cypress-realworld/bankaccounts.spec.js": "bbf9dc4f-9d87-4b7e-89e8-1b21e8570de4",
+  "flake/adding-spec.ts": "a8ebd71b-2acf-4c65-9c35-df110924c71c"
```

Comment:
Where are the source files for these two new tests?

They should be in `public/test/examples/` so we can regenerate them if necessary.

Reply:
Ah. Can you drop some documentation somewhere (inline in the examples JSON? in the README?) explaining how we are supposed to update these then? I wouldn't know.